projectPage.html

```diff
@@ -6,50 +6,38 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <meta name="HandheldFriendly" content="true">
+    <link href="projectPage.css" rel="stylesheet" type = "text/css">
     <title>PROJECTS</title>
 </head>
 
-<!-- <div class="all"> -->
-
 <body class="all" style="background: black;">
-
-<div class="vC"> 
-
-<header>
-    <video class="header-vid" playsinline autoplay muted loop>
-        <source src="atlB&W.mp4" type="video/mp4">
-    </video>
-    <h1 class="header-h1">Projects</h1>
-</header>
-
-<!-- <div class="row">  -->
-
-<div class="column">
-    <img src="cPic.jpg" alt="" style="width: 200px; height: auto;">
-</div>
-
-<div class="E" style="height: 10px;"></div>
-
-<div class="qP">
-    <h2 class="project-header">JavaScript Quiz</h2>
-    
-  
-    <video width="800px" height="550px" playsinline muted controls>
-        <source src="quizPreview.mp4" type="video/mp4" class="QV">
-      </video>
-      
-
-      <!-- <button class="btn success">Success</button> -->
-      <div class="qB">
-      <div class="Quizb">
-      <a href="https://johnsonce3101.github.io/JSQuiz/" class="btn" > Take The Quiz</a>
-    </div>
-</div>
-
-<div>
```

Comment:
cleaned up your indents, you want nested elements to be aligned correctly along indentations. easier to read.